### PR TITLE
Version bump v6.0.0

### PR DESCRIPTION
**Pull request by martinamarien**
### Purpose
Bump Dawn version to v6.0.0

### Approach 
* Updated settings schema version number
* Updated release notes

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -1,29 +1,15 @@
...
-- We’re now providing improved information on how to create meaningful alt text in the video section. 
-- We fixed an issue with making products full width on featured collections. 
 
-### Removed
-- In the collection and search templates, the “Collapse on desktop” checkbox has been removed from the filtering and sorting settings. If you’d like to achieve the collapsed style, select “Drawer” from the “Desktop filter layout” dropdown. 
+- On larger screen sizes, the quick add modal included unnecessary empty space. This is now fixed. 
+- We fixed an issue where the product media gallery sometimes hid the subsequent media when larger values of horizontal grid space were used. 
+- The product media gallery included unnecessary empty space between the product media and the gallery controls. This is now fixed.
+- We fixed an issue where the collapsible content section and the collapsible content block were missing bottom padding within each collapsible tab. 
+- We fixed an issue where hiding the  background of the content container on the slideshow section would not work if a gradient was set as the background color.
```
Comment (ludoboludo) [inline release-notes.md:14]:
```suggestion
- We fixed an issue where hiding the background of the content container on the slideshow section would not work if a gradient was set as the background color.
```

File: release-notes.md [hunk 2]
```diff
@@ -1,29 +1,15 @@
...
-- You can now enable a quick add button on your product cards, allowing your customers to quickly add items to their cart. Find this setting in the collection template and the featured collection section. 
-- The featured collection section is now more flexible. You can control the description, text styles, button styles, and layout in the section settings. 
-- You can now adjust how many products to show in the product recommendations section, allowing you more control over the layout of your store. 
-- The theme’s product filtering functionality now supports boolean (true or false) custom filters. 
+- We introduced a new cart behavior setting offering the options to surface a cart drawer, a pop up notification, or link directly to the cart page upon adding an item to cart. 
+- The rich text section is now more flexible. You can now customize the desktop content position, the content alignment, the size and formatting of headings, and the number of blocks added to the section.
+- We introduced new settings to the header section to control  top and bottom padding.
```
Comment (ludoboludo) [inline release-notes.md:6]:
```suggestion
- We introduced new settings to the header section to control top and bottom padding.
```

**Review by ludoboludo — APPROVED** [review]

File: release-notes.md [hunk 3]
```diff
@@ -1,29 +1,15 @@
...
-- You can now enable a quick add button on your product cards, allowing your customers to quickly add items to their cart. Find this setting in the collection template and the featured collection section. 
-- The featured collection section is now more flexible. You can control the description, text styles, button styles, and layout in the section settings. 
-- You can now adjust how many products to show in the product recommendations section, allowing you more control over the layout of your store. 
-- The theme’s product filtering functionality now supports boolean (true or false) custom filters. 
+- We introduced a new cart behavior setting offering the options to surface a cart drawer, a pop up notification, or link directly to the cart page upon adding an item to cart. 
+- The rich text section is now more flexible. You can now customize the desktop content position, the content alignment, the size and formatting of headings, and the number of blocks added to the section.
+- We introduced new settings to the header section to control top and bottom padding.
```
Comment (LucasLacerdaUX) [inline release-notes.md:6]: Should we add a note that this may change the default look for this section?

File: release-notes.md [hunk 4]
```diff
@@ -1,29 +1,15 @@
-Dawn v5.0.0 introduces new ways to customize the look and feel of your store with a vertical product filter layout, mega menu header style, and a quick add button option. This update also features fixes and improvements, including several accessibility enhancements. 
+Dawn 6.0.0 introduces a new cart drawer feature, additional customization options on the rich text section, and various fixes and enhancements.
```
Comment (LucasLacerdaUX) [inline release-notes.md:1]:
```suggestion
Dawn 6.0.0 introduces a new cart drawer feature, additional customization options on the rich text section, and various fixes and enhancements.
```

**Review by ludoboludo — APPROVED** [review]

**Review by sofiamatulis — APPROVED** [review]

File: release-notes.md [hunk 5]
```diff
@@ -1,29 +1,15 @@
...
```
Comment (sofiamatulis) [inline release-notes.md:1]: Small nitpick: `v6.0.0` ?

**Review by LucasLacerdaUX — APPROVED** [review]

**Pull request merged**